hokusai/config.yml

```diff
@@ -1,2 +1,6 @@
 ---
 project-name: frequency
+hokusai-required-version: ">=0.5.9"
```

Comment:
do we want to bump to current hokusai version (0.5.13)?

Reply:
I'd specify the minimum version that is actually required by the features in this configuration.

Reply:
okay. let's keep it at 0.5.9.